admin.py

```
@@ -2,19 +2,38 @@
+router = APIRouter()
 templates = Jinja2Templates(directory="templates")
 
-pw = b'$2b$12$9BhNNcsswdTbqEOI6qUI3.6DOwzFS.ZIRcEn9nSIFlXKRK5qPVxwO'
+load_dotenv()
+pw = os.getenv("PASSWORD").encode()
+
+TOKEN_LIFETIME = timedelta(hours=1)
+active_tokens = {} # token: expiry
+
+def verify_token(token: str | None) -> bool:
+    if not token: return False
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)